mapper/cumulocity/c8y_translator_lib/src/json.rs

```diff
@@ -0,0 +1,275 @@
+//! A library to translate the ThinEdgeJson into C8yJson
+//! Takes thin_edge_json bytes and returns c8y json bytes
+//!
+//! ```
+//! use c8y_translator_lib::json::CumulocityJson;
+//! let single_value_thin_edge_json = r#"{
+//!        "time": "2020-06-22T17:03:14.000+02:00",
+//!        "temperature": 23,
+//!        "pressure": 220
+//!     }"#;
+//! let output = CumulocityJson::from_thin_edge_json(
+//!             &String::from(single_value_thin_edge_json).into_bytes());
+//! ```
+
+use crate::serializer;
+use chrono::prelude::*;
+use thin_edge_json::{
+    json::{ThinEdgeJson, ThinEdgeValue},
+    measurement::GroupedMeasurementVisitor,
+};
+
+#[derive(Debug, Eq, PartialEq)]
+pub struct CumulocityJson;
```

Comment:
What's the purpose of this struct here? Is it only to hold the associated functions below? I would remove it and make these methods just module functions.